examples/example.css

```
@@ -0,0 +1,101 @@
+.test {
+    color: auto;
+    background-color: auto;
+    border-color: auto;
+    border: auto;
```

Biome (1.9.4)

[error] Unexpected shorthand property border after border-color



(lint/suspicious/noShorthandPropertyOverrides)